packages/opentelemetry-exporter-stackdriver-trace/src/trace.ts

```diff
@@ -0,0 +1,144 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ExportResult } from '@opentelemetry/base';
+import { NoopLogger } from '@opentelemetry/core';
+import { ReadableSpan, SpanExporter } from '@opentelemetry/tracing';
+import { Logger } from '@opentelemetry/types';
+import { GoogleAuth } from 'google-auth-library';
+import { google } from 'googleapis';
+import { StackdriverExporterOptions } from './external-types';
+import { getReadableSpanTransformer } from './transform';
+import { Span, SpansWithCredentials } from './types';
+
+google.options({ headers: { 'x-opentelemetry-outgoing-request': 1 } });
```

Comment:
can this `x-opentelemetry-outgoing-request` be a defined const variable ?

Reply:
Is there some reason? This is the only place this constant exists in the entire package

Reply:
It is generally common practice , it's the same as with "magic numbers".  `x-opentelemetry-outgoing-request` is already used in few other packages and it has been delcared there always as  `OT_REQUEST_HEADER`.

Reply:
Sure. We should probably move constants like this to some central location. Possibly the types repo